modules/docs/arrow-docs/docs/docs/patterns/optics/README.md

```diff
@@ -6,4 +6,26 @@ permalink: /docs/patterns/optics/
 
 ## Optics
 
-TODO
\ No newline at end of file
+TODO
+
+### Generating optics
```

Comment:
This doesn't really belong under patterns as this is rather a feature of `arrow-optics`. Shall we place this under `docs/optics/syntax` or `docs/optics/dsl/` and refer to it from `docs/optics/setter` @raulraja ?

Reply:
I think it's fine under `syntax` in the optics section.